video/out/gpu/hwdec.c

```diff
@@ -279,6 +281,12 @@ void ra_hwdec_ctx_init(struct ra_hwdec_ctx *ctx, struct mp_hwdec_devices *devs,
             if (strcmp(type, drv->name) == 0) {
                 load_add_hwdec(ctx, devs, drv, false);
                 break;
+            } else if (strcmp(type, drv->legacy_name) == 0) {
+                MP_WARN(ctx, "gpu-hwdec-interop was selected with the legacy name '%s'. "
+                             "Please it change to '%s' in your config file or command line.\n",
```

Comment:
```suggestion
                             "Please change it to '%s' in your config file or command line.\n",
```

Reply:
Whoops. I made the fix but forgot to squash it.